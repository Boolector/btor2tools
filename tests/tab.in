
; comment followed by new line

1 zero 32

; next line starts with a tab
	
