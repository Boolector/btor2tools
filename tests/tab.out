*** catbtor: parse error in 'tab.in' line 7: unexpected tab character at start of line
